lib/sync-DataSetModel.js

```diff
@@ -527,40 +510,19 @@ var self = {
   toJSON: function (dataset_id, returnData, cb) {
```

Comment:
@david-martin I think it's better to keep the `toJSON` method as it can be useful for debugging purpose.